packages/destination-actions/src/destinations/dawn/identifyUser/index.ts

```diff
@@ -0,0 +1,54 @@
+import type { ActionDefinition, RequestClient } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { DawnIdentifyUser } from '../dawn-types'
+
+const getEventFromPayload = (payload: Payload): DawnIdentifyUser => {
+  const identifyUserPayload: DawnIdentifyUser = {
+    user_id: payload.user_id || '',
+    traits: payload.traits || {}
+  }
+  return identifyUserPayload
+}
+
+const processData = async (request: RequestClient, settings: Settings, payload: Payload[]) => {
+  const identifyUsers = payload.map((value) => getEventFromPayload(value))
+  return request('https://api.dawnai.com/segment-identify', {
+    method: 'post',
+    json: identifyUsers,
+    headers: {
+      authorization: `Bearer ${settings.writeKey}`
+    }
+  })
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Identify User',
+  description: '',
```

Comment:
```suggestion
  description: '<Add description here of how this event will be interpretted downstream in Dawn>',
```

Can't have an empty string for description^.